packages/manager/cypress/e2e/core/kubernetes/lke-update.spec.ts

```diff
@@ -1003,11 +1003,22 @@ describe('LKE cluster updates', () => {
     });
 
     it('can add a node pool with an update strategy on an LKE enterprise cluster', () => {
-      const cluster = kubernetesClusterFactory.build({
+      const clusterRegion = extendRegion(
+        regionFactory.build({
+          capabilities: ['Linodes', 'Kubernetes', 'Kubernetes Enterprise'],
+          id: 'us-east',
+        })
+      );
```

Comment:
Is the `extendRegion` necessary here? I think we may be able to remove it. 

```suggestion
      const clusterRegion = regionFactory.build({
        capabilities: ['Linodes', 'Kubernetes', 'Kubernetes Enterprise'],
        id: 'us-east',
      });
```